site/docs/v1/tech/apis/users.adoc

```diff
@@ -762,6 +762,24 @@ The link:/docs/v1/tech/reference/data-types/#instants[instant] this login occurr
 [field]#logins``[x]``.ipAddress# [type]#[String]#::
 The IP address if provided during the login request.
 
+[field]#logins``[x]``.location.city# [type]#[String]# [since]#Available since 1.30.0#::
+The city where the login request originated.
+
+[field]#logins``[x]``.location.country# [type]#[String]# [since]#Available since 1.30.0#::
+The country where the login request originated.
+
+[field]#logins``[x]``.location.latitude# [type]#[Double]# [since]#Available since 1.30.0#::
```

Comment:
Above in the example JSON response you added `displayString` but omitted it here. It is technically a synthetic field, but it looks like. I didn't add  `@JSONIgnore` on it - so it is coming back in the response. This is probably ok, but if we are going to document it in the JSON we should probably add it here as well.